components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/appcontroller/ProxyApplicationCreator.java

```diff
@@ -94,7 +97,8 @@ public ProxyApplicationCreator() throws AppManagementException {
      *                                Throws this when policy id is failed while requesting
      *                                Throws this when failed to create,publish or subscribe web application
      */
```

Comment:
Java doc comment issue has fixed in https://github.com/wso2/carbon-appmgt/pull/539/files